src/css/main.scss

```diff
@@ -999,7 +999,7 @@ div.crossed-out .table-inner-row:before {
   }
 
   &::-webkit-scrollbar-thumb {
-    background-color: var(--blue-300);
+    background-color: var(--blue-301);
```

Comment:
Is the `301` a typo?

Reply:
It is not  a typo, it is a [different color](https://github.com/openboxes/openboxes/blob/feature/ui-redesign/src/css/colors.scss#L40)
Unfortunately this color was not specified on the color pallet on **figma** so I named it myself. 
Named it `blue-301`  because it is very similar to already defined color `blue-300`